packages/blade/src/components/Drawer/Drawer.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import type { Meta, StoryFn } from '@storybook/react';
+import { Drawer, DrawerBody, DrawerHeader } from './';
+import type { DrawerProps } from './';
+import { getStyledPropsArgTypes } from '~components/Box/BaseBox/storybookArgTypes';
+import { Box } from '~components/Box';
+import iconMap from '~components/Icons/iconMap';
+import { Button } from '~components/Button';
+import { AnnouncementIcon, DownloadIcon } from '~components/Icons';
+import { Card, CardBody } from '~components/Card';
+import { Badge } from '~components/Badge';
+
+export default {
+  title: 'Components/Drawer',
+  component: Drawer,
+  tags: ['autodocs'],
+  argTypes: {
+    ...getStyledPropsArgTypes(),
+    icon: {
+      name: 'icon',
+      type: 'select' as 'string',
+      options: Object.keys(iconMap),
+      mapping: iconMap,
+    },
+    _isVirtuallyFocussed: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+} as Meta<DrawerProps>;
+
+const DrawerTemplate: StoryFn<typeof Drawer> = () => {
+  const [isDrawerOpen, setIsDrawerOpen] = React.useState(false);
+  const [isSecondDrawerOpen, setIsSecondDrawerOpen] = React.useState(false);
+  return (
+    <Box>
+      <Button onClick={() => setIsDrawerOpen(!isDrawerOpen)}>Toggle Drawer</Button>
+      <Drawer isOpen={isDrawerOpen} onDismiss={() => setIsDrawerOpen(false)}>
+        <DrawerHeader
+          leading={<AnnouncementIcon size="large" />}
+          title="Announcements"
+          titleSuffix={<Badge color="positive">New</Badge>}
+          subtitle="This is an announcement"
+          trailing={<Button icon={DownloadIcon} />}
+        />
+        <DrawerBody>
+          <Card padding="spacing.0">
+            <CardBody>
+              <img
+                width="100%"
+                style={{
+                  aspectRatio: '2 / 1',
+                }}
+                alt="dancing cat"
```

Comment:
We love dancing cat, but he has to go 😢 let's add contextual examples.

Reply:
no

Reply:
ok kidding. ignore everything regarding documentation in this PR. I have already added contextual examples in my docs PR